packages/components/organisms/f-content-cards/src/tenants/es-ES.js

```diff
@@ -5,5 +5,13 @@ export default {
     loggedInTitle: 'Te mereces un capricho',
     loggedInSubtitle: 'Estas ofertas son personalizadas para ti, demasiado buenas para perdérselas.',
     loggedInTermsText: 'Términos y condiciones aquí',
-    loggedInTermsUrl: '/info/terminos-y-condiciones#ii.terminos-y-condiciones-de-los-cupones-promocionales-just-eat'
+    loggedInTermsUrl: '/info/terminos-y-condiciones#ii.terminos-y-condiciones-de-los-cupones-promocionales-just-eat',
+    stampCardStatus: {
+        0: 'Zero stamps earned out of five.',
```

Comment:
I noticed that for 0 and 5 we have english translation for all the tenants. And for es all stampCardStatuses are in english. Will these come later?

Reply:
Yes these are coming later, we just don't have these atm